docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+# Command-Line Tutorial
```

Comment:
Given that HELM is all command-line, does this need to be explicitly command-line or can it just be tutorial?

Reply:
Just "tutorial" is fine for now... I'm envisioning some sort of Python library API tutorial eventually, but I'm not sure what we want to do for that.

Will do the file rename in a separate commit to retain comment history.